signalfx-codahale/src/main/java/com/signalfx/codahale/SfxMetrics.java

```diff
@@ -186,6 +188,24 @@ public Histogram histogram(String metricName, Map<String, String> dimensions) {
         return build(MetricBuilder.HISTOGRAMS, metricName, dimensions);
     }
 
+    /**
+     * Get or create a new histogram with {@link ResettingExponentiallyDecayingReservoir} that
+     * clears the resorvoir after each {@link Reservoir#getSnapshot()}. Clearing resorvoir after
+     * snapshot removes the effect of old data points and provides better visibility (where needed).
+     *
+     * @param metricName
+     *         The metric name.
+     * @param dimensions
+     *         Additional dimension key/value pairs, as a map.
+     * @return The {@link Histogram} instance.
+     */
+    public Histogram resettingHistogram(String metricName, Map<String, String> dimensions) {
```

Comment:
Can you also provide the variant with `String... dimensions` ?

Reply:
Done